### ISSUE TITLE
Update okta-auth-java to use okta-sdk-java 2.0

### ISSUE THREAD

**Issue by venuduggireddy** [okta/okta-auth-java#68]
Getting following exception when migrating to okta-sdk-java 2.0.

```xml
<okta.version>2.0.0</okta.version>
<okta.auth.version>1.0.0</okta.auth.version>
```

```
Caused by: java.lang.NoClassDefFoundError: com/okta/sdk/lang/Classes
	at com.okta.authn.sdk.client.AuthenticationClients.builder(AuthenticationClients.java:43) ~[okta-authn-sdk-api-1.0.0.jar:1.0.0]
	at com.example.demo.OktaClientService.setup(OktaClientService.java:81) ~[classes/:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_241]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[na:1.8.0_241]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[na:1.8.0_241]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[na:1.8.0_241]
```

**Comment by bdemers** [comment]
@venkateshkatti-okta this is expected at the moment, but it is something we are currently working on.

(there were a few breaking changes in v2, so it is NOT compatible at the moment.

**Comment by arvindkrishnakumar-okta** [comment]
Will address this.

**Comment by vekdeq** [comment]
Hi, when can we start migrating to SDK 2.0 - as we continue to observe breaking point such as  Caused by: java.lang.ClassNotFoundException: com.okta.sdk.client.Proxy. Further analysis, I am unable to find this class in commons api - I may have missed it - if you know the jar, I can include it - please guide. 

Vivek Bedekar

**Comment by arvindkrishnakumar-okta** [comment]
@vekdeq : `Proxy` class now lives in `okta-commons-java` module here - https://github.com/okta/okta-commons-java/blob/master/http/http-api/src/main/java/com/okta/commons/http/config/Proxy.java.

**Comment by vekdeq** [comment]
Hay Arvind, thanks for replying but - the package name is different than com.okta.spring.boot.sdk.OktaSdkConfig in spring boot sdk 1.4.0 - it still refers to old proxy package. Hence it breaks in app deployment - when I migrated to boor starter 1.4.0 & sdk & auth 2.0.0

Once again - thanks for quick reply. Stay safe. 

Vivek

**Comment by arvindkrishnakumar-okta** [comment]
@vekdeq We've not updated the spring boot sdk yet. It is in the works.

**Comment by vekdeq** [comment]
Got it - thanks .. So we can't upgrade yet.

**Comment by serega-upgini** [comment]
Hi! Is there any workaround here? I have no idea how to solve it :)

**Comment by bdemers** [comment]
Sorry we didn’t update this issue.
We released v2.0.0 of okta-auth-java which  resolves this issue.
